src/test/java/use_cases/create_new_month/NewMonthUCITest.java

```diff
@@ -18,8 +18,11 @@ class NewMonthUCITest {
     static MonthlyStorage monthlyStorage1;
     static SessionStorage sessionStorage;
 
+    /**
+     * @throws EntityException if an error occur with addMonth() in SessionStorage
```

Comment:
You are missing a comment explaining what the method does!

Reply:
Updated!